config/drupal_modules_disable.sh

```diff
@@ -7,7 +7,6 @@
 
 MODULES_DISABLE=(
   "color"
-  "overlay"
```

Comment:
@ordavidil 
It's default in our installation script because we really don't like to use overlay.
If it is necessary for the project (as I guess for the library upload and stuff), please make 100% sure the functionality is working on mobile.